compositions/upbound-aws-provider/aurora/aurora.yaml

```diff
@@ -0,0 +1,551 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: xauroras.db.awsblueprint.io
+  labels:
```

Comment:
Is this supposed to be for Postgres only? If so, the name and labels should reflect that.

Reply:
We should use the sam composition for both  flavour. But I have not tested the mysql . I will do that and add a claim in near . If needed I can add another issue for this enhancement. Please suggest.

Reply:
updated as discussed.